util/mod_translation_updater.py

```diff
@@ -121,17 +120,37 @@ def main():
 
 # Group 2 will be the string, groups 1 and 3 will be the delimiters (" or ')
 # See https://stackoverflow.com/questions/46967465/regex-match-text-in-either-single-or-double-quote
-pattern_lua_quoted = re.compile(r'[\.=^\t,{\(\s]N?F?S\s*\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)[\s,\)]', re.DOTALL)
+pattern_lua_quoted = re.compile(
+	r'(?:^|[\.=^\t,{\(\s])' # Look for beginning of file or anything that isn't a function identifier
+	r'N?F?S\s*' # Matches S, FS, NS or NFS function name followed by optional spaces
+	r'\(\s*(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)' # Quoted string
```

Comment:
this makes more sense is more accurate IMO
```suggestion
	r'N?F?S\s*\(\s*' # Matches S, FS, NS or NFS function call
	r'(["\'])((?:\\\1|(?:(?!\1)).)*)(\1)' # Quoted string
```